yarn.lock

```diff
@@ -19145,16 +19145,16 @@ passport-onelogin-oauth@^0.0.1:
     pkginfo "0.2.x"
     uid2 "0.0.3"
 
-passport-saml@^1.3.3:
-  version "1.3.3"
-  resolved "https://registry.npmjs.org/passport-saml/-/passport-saml-1.3.3.tgz#cbea1a2b21ff32b3bc4bfd84dc39c3a370df9935"
-  integrity sha512-54ecY/A6UEsyCehJws6a+J6THvwtYnGl9cnAUxx5DjsuKgZrDs0tSy58K4hCk1XG/LOcdQSF1TR3xlRXgTULhA==
+"passport-saml@^1.3.5, <1.4.0":
```

Comment:
Newer versions >= 1.4.0 use TypeScript, and until https://github.com/node-saml/passport-saml/pull/516 is released we can't use the new versions; tsc won't pass.

Reply:
Who knew the pain of migrating to TS would not be introducing types to the NPM ecosystem, but to replace the initial types with first party ones 😅